### DISCUSSION TITLE
Feature Suggestion- Copy hex value to clipboard on eyedropper click

### DISCUSSION THREAD

**Discussion by mostlyhuman** [Toinane/colorpicker#93] [Ideas]
When you use the eyedropper to click a color on your screen, it would be helpful if it automatically copied the hex value to the clipboard.  As is I have to highlight the hex value in the tool then copy it to the clipboard.  Cheers!

**Comment by Mineinjava** [comment]
a right click might be better instead of a left click for copying the hex color code
